Dockerfile.standalone

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:16.04
+
+#---install useful tools and dependencies---#
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
+  jq curl vim nano ruby sudo openssh-client
+# as per https://hub.docker.com/r/conjurinc/cli5/~/dockerfile/
+
+#---install Conjur 4 CLI---#
+RUN curl -sSL -o conjur-cli.deb \
+  https://github.com/cyberark/conjur-cli/releases/download/v5.4.0/conjur_5.4.0-1_amd64.deb
+RUN dpkg -i conjur-cli.deb
+RUN apt-get install -qyf
+COPY standalone.entrypoint /bin/entry
+
+#---install summon and summon-conjur---#
+ENV CONJUR_MAJOR_VERSION=4
+ENV CONJUR_VERSION=4
+RUN curl -sSL https://raw.githubusercontent.com/cyberark/summon/master/install.sh \
+  | env TMPDIR=$(mktemp -d) bash
```

Comment:
Why does `TMPDIR` need to be set here?

Reply:
If you don't set a different TMPDIR for the two installs, they conflict and the second install (summon-conjur) fails. That should probably be fixed in the `summon` and `summon-conjur` install scripts, but this is a pretty straightforward workaround.